tests/h/search/core_test.py

```diff
@@ -0,0 +1,243 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import datetime
+
+import pytest
+
+from h import search
+
+
+class TestWithoutSeparateReplies(object):
+    """Tests for what happens when no separate_replies argument is given."""
+
+    def test_it_returns_replies_in_annotations_ids(self, matchers, pyramid_request, Annotation):
+        """Without separate_replies it returns replies in annotation_ids.
+
+        Test that if no separate_replies argument is given then it returns the
+        ids of replies mixed in with the top-level annotations in
+        annotation_ids.
+        """
+        annotation = Annotation(shared=True)
+        reply_1 = Annotation(shared=True, references=[annotation.id])
+        reply_2 = Annotation(shared=True, references=[annotation.id])
+
+        result = search.Search(pyramid_request).run({})
```

Comment:
It's kind of annoying that you still have to pass `{}` to `run()` to do a search with no params.